packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2737,6 +2737,8 @@ export class ContainerRuntime
 				return { continue: true };
 			};
 
+			this._summarizer?.markSummaryAttempt?.(summaryRefSeqNum);
```

Comment:
not sure if it matters but do we want to record the attempt  even when we are not going to continue ?

Reply:
The `SummaryGenerator` records the attempt if this method returns anything, and it seems to return in most cases